package.json

```diff
@@ -9,17 +9,26 @@
     "autoformat": "npx prettier -w ."
   },
   "dependencies": {
+    "@react-native-community/async-storage": "^1.12.1",
+    "@react-native-community/masked-view": "0.1.10",
+    "@react-navigation/native": "^5.9.2",
+    "@react-navigation/stack": "^5.14.2",
     "expo": "~40.0.0",
     "expo-status-bar": "~1.0.3",
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-40.0.1.tar.gz",
-    "react-native-web": "~0.13.12"
+    "react-native-gesture-handler": "^1.10.1",
+    "react-native-reanimated": "~1.13.0",
```

Comment:
I'm not a fan of the dependency versioning mode mix (`"0.1.10"`, ` "^5.9.2"`, `"~1.13.0"`) ... Is there a reason to do it like that ?

Reply:
It's due to `npm audit fix` command which force "version" to one with no breaches.